packages/quantic/coveo.d.ts

```diff
@@ -2,20 +2,31 @@
 import * as HeadlessTypes from './force-app/main/default/staticresources/coveoheadless/definitions/index';
 export * from './force-app/main/default/staticresources/coveoheadless/definitions/index';
 import {LightningElement} from 'lwc';
+import {Deferred} from 'utils';
+import {CoreEngine} from './force-app/main/default/staticresources/coveoheadless/definitions/app/engine';
+
+interface Bindings {
```

Comment:
Bindings object as used in [atomic](https://github.com/coveo/ui-kit/blob/master/packages/atomic/src/utils/initialization-utils.tsx).
This contains:
- a reference to the engine instance
- a reference to i18n object that will be used for localization.
- a reference to a flexible "store" object to store data that can be used by components.

The i18n reference isn't set as it isn't yet implemented (see [this jira](https://coveord.atlassian.net/browse/SFINT-3962))

Reply:
For i18n, I would make sure it is "synch" with whatever Salesforce is doing for language management.

AKA: I'm guessing there's a way in a salesforce community to manage different languages. It should be reflected in quantic when an administrator make those changes.

And not have it be completely disconnected, if possible.

Reply:
Yes you're correct. We have a separate jira for implementing localization. I thought i'd leave this here as a starting point for whoever implements it but i'll remove it instead so we don't have a dangling object that might not be used.